src/main/java/at/molindo/notify/dispatch/AbstractPushDispatcher.java

```diff
@@ -190,6 +190,8 @@ protected void dispatch(@Nonnull Notification notification) {
 	 *             only if ignoreFrequency is true
 	 */
 	private void dispatch(Notification notification, boolean ignoreFrequency) throws NotifyException {
+		// XXX: shouldn't that be push(notification, ignoreFrequency) instead of
+		// push(notification, true)?
```

Comment:
it probably should. non-instant notifications (e.g. weekly digest) have never been implemented though, thus this bug never had an impact